template/page-docs.html.jinja

```diff
@@ -129,12 +129,13 @@
 
 {% block endbody %}
 <script type="text/javascript"> docsearch({
-apiKey: '70de5b9b8ef95460f3c6e04054b325b2',
-indexName: 'xrpl',
-inputSelector: '#docsearchbox',
-algoliaOptions: { 'facetFilters': ["lang:{{target.lang}}"] },
-debug: false
-});
+  apiKey: '3431349deec23b0bc3dcd3424beb9a6e',
+  indexName: 'xrpl',
+  container: '#centersearchboxcontainer',
+  appId: 'R39QY3MZC7',
+  algoliaOptions: { 'facetFilters': ["lang:en"] },
```

Comment:
Looks better now, but I noticed that one of my first searches took me to a page that's in the Japanese target (but presented in English because it's not translated yet). I'm not sure if that's a bug on Algolia's side or if there's something wrong with how this option is given. Maybe it's because this is only for the `#centersearchboxcontainer` and not the `#topsearchboxcontainer`?

Also, from the looks of it, this will always search English even if the user is on the Japanese version of the site, which is not ideal. The syntax in the old diff (using `{{target.lang}}`) should still work here.

Reply:
What are you searching? when searching EVM it was giving me Japanese at first too but now the results are in English. Also the preview with "/ja" doesn't really work - can't tell if it's working or not.